src/main/resources/static/js/alert.js

```diff
@@ -0,0 +1,7 @@
+$(function() {
```

Comment:
...I took a quick look and didn't find a webjar for it. Is there an NPM/Bower for it?

In any case, prune the cruft and place in a sub-directory will be good enough.